.circleci/config.yml

```diff
@@ -2,12 +2,24 @@ version: 2
 
 workflows:
   version: 2
-  full_test:
+  test_and_release:
     jobs:
+      - create_release_pull_request:
+          filters:
+            branches:
+              only:
+                - /^Version-v(\d+)[.](\d+)[.](\d+)/
```

Comment:
This only runs upon merge of a version branch?

Reply:
This instructs CircleCI to only run the `create_release_pull_request` job on branches matching this pattern. That is, any commits to a version branch will trigger this job.